examples/subproc2.py

```diff
@@ -3,6 +3,6 @@
 import sys
 
 num = int(sys.argv[1])
-for c in xrange(1,10000000):
+for c in range(1,10000000):
```

Comment:
really, really don't want this in python2 :-) maybe an explicit check for PYTHON2 is in order